components/index/carousel.js

```diff
@@ -66,7 +58,7 @@ export default function Carousel() {
                   titleColor="#FF4794"
                   descriptionColor="white"
                   title="Epoch"
-                  description="Attend the most epic high school hackathon this New Year's"
+                  description="Attend the most epic high school hackathon this New Year’s"
                   img="https://a.slack-edge.com/production-standard-emoji-assets/14.0/apple-large/1f386@2x.png"
                   link="https://epoch.hackclub.com"
```

Comment:
might wanna remove this

Reply:
maybe /events? it shouldn't be front-and-center on the homepage imo since its done

Reply:
talking of /events, we need a page to show the grand hackclub-organized events other than events.

Reply:
oooooooooo that could be pretty cool! i think it would also be great if we could find a way to work in language about hack club's incredible summer (and maybe now even winter) events :)

Reply:
agreed that for now it feels out of context though, will remove :)

Reply:
imho before that's done THF/HC needs to decide if hack club summers are gonna be a continuing thing onwards

then we can get craftin'

Reply:
i think for now we can assume so... but before we merge i'll get final sign off from zach/christina :)

Reply:
do you have any suggestions?

Reply:
just got off the pc but i have a _general_ wireframe in my head

we have photo&waivers from hack camp right?

Reply:
tracked in #670

Reply:
wahoo, thanks! i believe so!